docs/UserGuide.md

```diff
@@ -147,6 +147,42 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Adding new Project : `addP`
+
+Adds a new project in TaskHub
+
+Format: `addP p/PROJECT_NAME [e/EMPLOYEE_NAME] ...`
+
+* Adds a new project with the employees assigned to the project.
+* The employee must exist in the contacts list. 
```

Comment:
Since it is no longer an AddressBook, but TaskHub, I don't think it's appropriate to call it `contacts list`. I think a more appropriate name is `employees list`.